src/components/radio-group/radio-group.ts

```diff
@@ -17,15 +17,63 @@ import styles from './radio-group.styles';
 @customElement('sl-radio-group')
 export default class SlRadioGroup extends LitElement {
   static styles = styles;
+  private _value: string;
 
   @query('slot:not([name])') defaultSlot: HTMLSlotElement;
 
   /** The radio group label. Required for proper accessibility. Alternatively, you can use the label slot. */
   @property() label = '';
 
+  /** The current value of the radio group. */
+  @property()
+  get value() {
```

Comment:
This might be odd, but I need a way to get the value after render, but not have it be blank because the default isn't in sync with the defaulted 'checked' radio. 

If you know a better way to ensure this has a good default when the user does a `get` on the value.... I'm open to it.